spec/controllers/api/v1/changesets_controller_spec.rb

```diff
@@ -19,7 +19,7 @@
     it 'returns a changeset when its ID is given' do
       create_list(:changeset, 2)
       get :show, id: Changeset.last.id
-      expect_json({
+      expect_json('changeset', {
```

Comment:
Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.